### PR TITLE
add german translation for v2.0.0

### PR DESCRIPTION
**Pull request by futape**

### CODE REVIEW

**Comment by haacked** [comment]
Thanks! Could you find another native German speaker to verify the translation? I'm not really capable of reviewing this. Danke!

**Comment by futape** [comment]
i'm going to find someone reviewing the translation in my personal environment. when done, i'll write here.

i'm not quite sure whether i understood the "What do I do if I accidentally release a backwards incompatible change as a minor version?" and the "What if I inadvertently alter the public API in a way that is not compliant with the version number change (i.e. the code incorrectly introduces a major breaking change in a patch release)" paragraphs right? could you please explain them to me? thanks.

**Comment by futape** [comment]
found someone checking my translation, may take up to 3 weeks.

**Comment by futape** [comment]
finally the translation has been reviewed by a friend of mine and a few typos have been fixed. now everything sould be properly set up to be merged.

File: lang/de/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+Zusammenfassung
+-------------
+
+Auf Grundlage einer Versionsnummer von MAJOR.MINOR.PATCH werden die einzelnen Elemente folgendermaßen erhöht:
+
+1.  MAJOR wird erhöht, wenn API-inkompatible Änderungen veröffentlicht werden,
+2.  MINOR wird erhöht, wenn neue Funktionalitäten, welche kompatibel zur bisherigen API sind, veröffentlicht werden, und
+3.  PATCH wird erhöht, wenn die Änderungen ausschließlich API-kompatible Bugfixes umfassen.
+
+Außerdem sind Bezeichner für Pre-Releases und Build Metadaten als Erweiterungen zum MAJOR.MINOR.PATCH Format verfügbar.
```
Comment (majewsky) [inline lang/de/index.md]: "Build Metadaten" -> "Build-Metadaten". Außerdem würde ich "prerelease" womöglich als "Vorveröffentlichung" übersetzen, da "release" immer als "Veröffentlichung" übersetzt wird.

Edit: Urgh, these are going to be a bit too many notes that I'm going to add. How can we do the proof-reading process in a nicer way? Should I submit a pull request against the fork's branch that this PR is based on?

**Comment by nekomajin** [comment]
This question I ask myself too. I'm working on a german translation too. So how do we stay?

**Comment by majewsky** [comment]
How about the following process?
1. We let @mojombo merge this one first (it still looks solid, notwithstanding any possible stylistic issues).
2. One of us sets up a fork where the others can commit, too, and we all commit our stylistic fixes to a feature branch there, in order to limit the noise that is generated over here.
3. Once we're satisfied with the results, we do another pull request.

**Comment by nekomajin** [comment]
In my opinion, a very good decision. I plan to provide a german translation for the past versions. For the sake of completeness. This we can also prepare and then do a pull request for each translation.

**Comment by haacked** [comment]
Hi all, @mojombo doesn't maintain this repo anymore. I'm happy to merge this, but it has conflicts that must be resolved. Afterwards, if you all agree this is a good enough translation, I'll merge it.

**Comment by futape** [comment]
i fixed the merge conflicts. regarding the _noise_: sorry if i messed anything up, but what exactly do you mean?

File: lang/de/index.md (hunk 1 above)
Comment (haacked) [inline lang/de/index.md]: I think adding line comments are fine for now. Let's see how many you add. A PR into this PR is also fine. Really up to the original submitter.

**Comment by haacked** [comment]
@futape there's [one comment by a reviewer](https://github.com/mojombo/semver.org/pull/117/files#r43771702) that hasn't been addressed.

**Comment by futape** [comment]
i thought this pr should be merged first (as it is) and someone else would do a new fork, adds the remaining changes and does a pr to this repo. but well, i will replace "Build Metadaten" with "Build-Metadaten" and "prerelease" with "Vorveröffentlichung".

**Comment by haacked** [comment]
> i thought this pr should be merged first (as it is) and someone else would do a new fork, adds the remaining changes and does a pr to this repo. but well, i will replace "Build Metadaten" with "Build-Metadaten" and "prerelease" with "Vorveröffentlichung".

Well, I wanted to get at least the existing review comments addressed. I'd rather avoid merging something known to be wrong. :smile:

**Comment by haacked** [comment]
Nice work! :+1: 

thumbsup-nod

**Pull request merged**